examples/pattern_matching_showcase/python/config.py

```diff
@@ -142,7 +142,7 @@ def s_to_ms(seconds):
 if len(sys.argv) > 1 and "TEST_MODE" in sys.argv[1:]:
     total_simulation_time = 20.0
 
-nest_n_threads = 1  # None -> auto; Integer -> according fixed number of threads
+nest_n_threads = 16  # None -> auto; Integer -> according fixed number of threads
```

Comment:
I guess we should stick to a lower value here. As a demonstration 2-4 should be totally sufficient.